TOOLS/dylib-unhell.py

```diff
@@ -116,6 +128,9 @@ def main():
     while libs_processed is not None:
         libs_processed = process_libraries(libs, binary, libs_processed)
 
+    print(">> removing rapths of dev tools ")
```

Comment:
`rapths` → `rpath` (commit message as well) and probably "removing rpath definitions towards dev. tools" or something?